packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -153,7 +155,9 @@ class CloudFormation {
   }
 
   readStackEvents(stackName) {
-    this.pollForEvents = setInterval(() => this.addToPollQueue(stackName, 3), CFN_POLL_TIME);
+    const waitTimeInMs = Math.min((2 ^ this.readStackEventsCalls) * CFN_POLL_TIME, CFN_POLL_TIME_MAX);
+    this.pollForEvents = setInterval(() => this.addToPollQueue(stackName, 3), waitTimeInMs);
+    ++this.readStackEventsCalls;
```

Comment:
@jhockett I reworked this part in https://github.com/aws-amplify/amplify-cli/pull/11624 in order to get searchable migration test passing.